### PR TITLE
Add ParentEntry to code editor

### PR DESCRIPTION
**Pull request by Skirlez**
## Description
This PR makes it so you can see the ParentEntry property in the code editor.
Often when searching for a specific function you'll find the code entry referencing it, but then have to manually search for the parent entry to view/edit its contents. This change would make it easier to go there, you just need to double click it.

The property won't be displayed when null (so it will only show up when relevant), and can't be edited or set to null.

Screenshot from Void Stranger's `gml_Script_scr_steam_achievement_get`:
image

### CODE REVIEW

**Comment by zivmaor** [comment]
Seems like a good addition.
One caveat with this is that you can change the ParantEntry by dragging a different one in, which makes no sense and doesn't update that entry's ChildEntries list, but it doesn't seem to cause any bugs I could find and it doesn't affect the saved data.win file so it might be fine to leave as is.

**Comment by colinator27** [comment]
> Seems like a good addition to me. One caveat with this is that you can change the ParantEntry by dragging a different one in, which makes no sense and doesn't update that entry's ChildEntries list, but it doesn't seem to cause any bugs I could find and it doesn't affect the saved data.win file so it might be fine to leave as is.

I think this can be addressed by adding a boolean to `UndertaleModTool/Controls/UndertaleObjectReference.xaml.cs`, specifying whether any changes (by drag and drop) are allowed. That or with some fancy bindings, but I'm not too sure how WPF handles that.

**Comment by Miepee** [comment]
@Skirlez can you send a new screenshot for GUI? Am interested in seeing a specific thing there.

File: UndertaleModTool/Controls/UndertaleObjectReference.xaml.cs [hunk 1]
```diff
@@ -94,8 +100,14 @@ public Type ObjectType
 
         public bool CanRemove
         {
-            get { return (bool)GetValue(ObjectTypeProperty); }
-            set { SetValue(ObjectTypeProperty, value); }
+            get { return (bool)GetValue(CanRemoveProperty); }
+            set { SetValue(CanRemoveProperty, value); }
+        }
+
+        public bool CanDragInto
```
Comment (Miepee) [inline UndertaleModTool/Controls/UndertaleObjectReference.xaml.cs:107]: This attribute feels like it should be "can be changed" instead.

**Review by Miepee — APPROVED** [review]
Hm, could maybe do something else UX wise to indicate more that its a read only field, but I can't think of anything right now.
Thanks for the PR!

**Pull request merged**